e2e/e2etests/e2etests.go

```diff
@@ -513,6 +507,12 @@ var AllE2ETests = []runner.E2ETest{
 		[]runner.ArgDefinition{},
 		TestRateLimiter,
 	),
+	runner.NewE2ETest(
+		TestERC20DepositAndCallRefundName,
```

Comment:
this test uses admin apis so it should run on the admin test runner

Reply:
Good catch.
Although the admin functionality used doesn't interfere with the network so would be fine to keep it in regular tests.

The idea of separating admin tests is that functions that interfere with network functionality (pausing cctxs for example) can't be run in parallel with the rest of the tests, hence running separately. And they also tests scoped functionalities in the protocol.

We might want to do some refactoring or renaming here, calling "admin special tests" for example.

The other thing with test using admin API is that it can be used for tests on live network. For this I think we should introduce a new boolean in the E2E test object to specify if the test can be run on live network or not.

Reply:
Ok I can move it back to the erc20 test runner, I will just need to inject the txserver into the runner.